src/common/sockman.cpp

```diff
@@ -9,68 +9,77 @@
 #include <netbase.h>
 #include <util/sock.h>
 
-bool SockMan::BindListenPort(const CService& addrBind, bilingual_str& strError)
+bool SockMan::BindAndStartListening(const CService& to, bilingual_str& err_msg)
```

Comment:
In commit "style: modernize the style of SockMan::BindListenPort()" (6ba0aab8658a7bd553a96e441aa1d5e016e086c3)

Could be good to note in commit message what behaviors this is changing, so it clear they are intentional.  After this commit:

- err_msg / strError output is no longer set if there are minor errors like failing to set SO_REUSEADDR
- "Bound to [address]" log message is dropped and moved to caller
- Other log messages are changed slightly